### DISCUSSION TITLE
64Box will be?

### DISCUSSION THREAD

**Discussion by Danik2343** [86Box/86Box#1891] [General]
Hi everyone.
Will there be a 64Box version with support for 64 bit processors, compatible with 86 applications, etc?

**Comment by ghost** [comment]
Soontm.

**Comment by Cacodemon345** [comment]
Only when graphene chips are a thing.

**Comment by evansstuff** [comment]
someone please remind me of this thread in 20 years

**Reply by jriwanek** [reply]
Please don't take random comments not made by the 86box team itself as promises, we have no plans at this time to emulate 64-bit CPU's

**Comment by TechGuyC** [comment]
> Hi everyone. Will there be a 64Box version with support for 64 bit processors, compatible with 86 applications, etc?

When 128-bits processors start to exist.
